Lib/test/test_capi/test_misc.py

```diff
@@ -2427,6 +2436,7 @@ def new_interp(config):
         with self.subTest('main'):
             expected = _interpreters.new_config('legacy')
             expected.gil = 'own'
+            expected.check_multi_interp_extensions = False
```

Comment:
I think this should be conditional:
```suggestion
            if not Py_GIL_DISABLED:
                expected.check_multi_interp_extensions = False
```

Reply:
I made the check conditional (but it needs to be `if Py_GIL_DISABLED:`)